### DISCUSSION TITLE
Automatic prop generation

### DISCUSSION THREAD

**Discussion by toerob** [toerob/vscode-tads3tools#24] [General]
I’m curious. Does the automatic prop generator have a real use case? I seldom use it myself but I think I might use it more if I saw it as a suggestion above an object/class description and could activate it by just clicking on the text. Not sure though if seeing these suggestions would only become distracting and what way to  best show/hide them to provide the best user experience.

(I’m hoping with this post to discuss how to further improve this feature and possible new features similar to it. )

**Comment by johnnywz00** [comment]
I think the idea is clever but I don't end up using the feature myself, for a particular reason. In my game I have several different classes that I use for "decoration" objects, apart from the standard Decoration class. I usually end up going through my room description and picking out the nouns one by one and deciding which form of "not important" class to give to it...

Did you see my additional info on the symbol outliner collapsing issue?
